setup.py

```diff
@@ -14,7 +14,7 @@
     tests_require=["pytest", "aiohttp", "pytest-asyncio", "aresponses"],
     long_description_content_type="text/markdown",
     url="https://github.com/ludeeus/pytraccar",
-    packages=setuptools.find_packages(),
+    packages=setuptools.find_packages(exclude=("tests",)),
```

Comment:
```suggestion
    packages=setuptools.find_packages(exclude=["tests.*", "tests"]),
```
in case there is directories in the in the future

Reply:
Sure, why not. Or just simply `tests*` would have worked as well.